packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 48.13.0 - April 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Add check for local dev server start failure ([PR #2128](https://github.com/forcedotcom/salesforcedx-vscode/pull/2128))
+
+## Added
+
+#### docs
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
+
+#### salesforcedx-vscode-core
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
+
+- Add diagnostics for beta deploys ([PR #2143](https://github.com/forcedotcom/salesforcedx-vscode/pull/2143))
```

Comment:
We should add a link to this issue that got resolved by this fix: https://github.com/forcedotcom/salesforcedx-vscode/issues/2124

Reply:
I would add that fix as a separate item and add the link Ananya mentioned.

Reply:
Added that and moved to the fixed section. Let me know if that's fine.